### DISCUSSION TITLE
Release Sprint 2022

### DISCUSSION THREAD

**Discussion by odhiaaa** [inlivedev/inlive-website#145] [Release Preparation]
Release Format
-----------------

# Deployment Guidelines
Sprint Date Year
Version vX.XX

## Checklist

- [ ] All PR have been merged to development branch
- [ ] Passed test and approved
   - [ ] All added features or functions have been tested in development environment
   - [ ] All fixed bugs or errors have been tested in development environment
   - [ ] All changed features or functions have been tested in development environment
   - [ ] Reviewed and approved by Engineering Manager
   - [ ] Approved by Product Manager
- [ ] Prepare draft Release Note
   - [ ] Create draft release from repository
   - [ ] Create a new tag based on this document subtitle
   - [ ] Set target to default branch
   - [ ] Fill the description, use this format
   - [ ] Save draft
- [ ] Do any prerequisite if any
- [ ] PR to production environment
   - [ ] Open PR from development branch to default branch, use this format 
   - [ ] Approved by Reviewer (1 or more)
   - [ ] Merged using Squash and Merge
- [ ] Publish draft Release Note
- [ ] Recheck Issues and PR
   - [ ] Close all Issues that related to this deployment and still open
   - [ ] Close all PR that related to this deployment and still open (duplicate, invalid, etc)

## PR Format

**Title:** 
Merge Dev to Main vX.XX

**Description:**
```
This will merge the current branch dev code into main.

## This PR consists of updates from these PRs:
- Number of PR
  - Close Number of issue

## Release notes:
TBA
```

## Release Note Format

**Title:** 
Release vX.XX

**Description:**
```
## Release notes:
TBA

## This version consists of updates from these PRs
- Number of PR
  - Close Number of issue
```

**Comment by ingrid011** [comment]
mba @odhiaaa & mas @faiq-naufal Sudah aku buatkan draft PR & draft release nya ya as per ini.. fyi.. please help to check again :pray:

**Reply by odhiaaa** [reply]
thankyou mba, tinggal execute aja nanti

**Comment by odhiaaa** [comment]
# Deployment Guidelines
Sprint June 2022
Version: v0.2.1

## Checklist

- [x] All PR have been merged to development branch
- [x] Passed test and approved
   - [x] All added features or functions have been tested in development environment
   - [x] All fixed bugs or errors have been tested in development environment
   - [x] All changed features or functions have been tested in development environment
   - [x] Reviewed and approved by Engineering Manager
   - [x] Approved by Product Manager
- [x] Prepare draft Release Note
   - [x] Create draft release from repository
   - [x] Create a new tag based on this document subtitle
   - [x] Set target to default branch
   - [x] Fill the description, use this format
   - [x] Save draft
- [x] Do any prerequisite if any
- [x] PR to production environment
   - [x] Open PR from development branch to default branch, use this format 
   - [x] Approved by Reviewer (1 or more)
   - [x] Merged using Squash and Merge
- [x] Publish draft Release Note
- [x] Recheck Issues and PR
   - [x] Close all Issues that related to this deployment and still open
   - [x] Close all PR that related to this deployment and still open (duplicate, invalid, etc)

## PR Format

**Title:** 
Merge Dev to Main v0.2.1

**Description:**
```
This will merge the current branch dev code into main.

## This PR consists of updates from these PRs:
- #140
  - Close #139,
  - Close #143,
  - Close #144

## Release notes:
- Added Interactive Widget announcement blog
- Added tutorial live chat widget in the doc
- Changed documentation to announce Interactive Widget
- Changed widget landing page content to announce the interactive widget
```

## Release Note Format

**Title:** 
Release v0.2.1

**Description:**
```
## Release notes:
- Added Interactive Widget announcement blog
- Added tutorial live chat widget in the doc
- Changed documentation to announce Interactive Widget
- Changed widget landing page content to announce the interactive widget

## This version consists of updates from these PRs
- #140
  - Close #139,
  - Close #143,
  - Close #144
```

**Comment by odhiaaa** [comment]
# Deployment Guidelines
Sprint August 2022
Version: v0.2.2

## Checklist

- [ ] All PR have been merged to development branch
- [ ] Passed test and approved
   - [ ] All added features or functions have been tested in development environment
   - [ ] All fixed bugs or errors have been tested in development environment
   - [ ] All changed features or functions have been tested in development environment
   - [ ] Reviewed and approved by Engineering Manager
   - [ ] Approved by Product Manager
- [ ] Prepare draft Release Note
   - [ ] Create draft release from repository
   - [ ] Create a new tag based on this document subtitle
   - [ ] Set target to default branch
   - [ ] Fill the description, use this format
   - [ ] Save draft
- [ ] Do any prerequisite if any
- [ ] PR to production environment
   - [ ] Open PR from development branch to default branch, use this format 
   - [ ] Approved by Reviewer (1 or more)
   - [ ] Merged using Squash and Merge
- [ ] Publish draft Release Note
- [ ] Recheck Issues and PR
   - [ ] Close all Issues that related to this deployment and still open
   - [ ] Close all PR that related to this deployment and still open (duplicate, invalid, etc)

## PR Format

**Title:** 
Merge Dev to Main v0.2.2

**Description:**
```
This will merge the current branch dev code into main.

## This PR consists of updates from these PRs:
- #153 
  - Close #150 
- #155 
  - Close #154 
_nb: team livestream please add more_

## Release notes:
- Added tutorial viewer counter widget on doc
- Added tutorial banner widget on doc
_nb: team livestream please add more_
```

## Release Note Format

**Title:** 
Release v0.2.2

**Description:**
```
## Release notes:
- Added tutorial viewer counter widget on doc
- Added tutorial banner widget on doc
_nb: team livestream please add more_

## This version consists of updates from these PRs
- #153 
  - Close #150 
- #155 
  - Close #154 
_nb: team livestream please add more_
```

**Reply by ingrid011** [reply]
@odhiaaa thank you mba. 
@odhiaaa @faiq-naufal @bernardsuryawan the [draft merge dev to main PR](https://github.com/asumsi/inlive-website/pull/158) & [draft release note](https://github.com/asumsi/inlive-website/releases/tag/untagged-8ce578fbebb1915fbdef) already made ya, kindly recheck it if there's any changes.

Thank you.
